i18n/locales/en/common.json

```diff
@@ -548,6 +547,7 @@
   "What is a Lisk ID?": "What is a Lisk ID?",
   "What is your passphrase?": "What is your passphrase?",
   "What's New...": "What's New...",
+  "Whoops, that page is gone.": "Whoops, that page is gone.",
```

Comment:
Do you know if this will be the text that we gonna use on this component? I mean I know this is how looks the design but, the text is weird. Maybe just confirm this with Ali to double check that this is it or not/

Reply:
@reyraa Can you confirm this text, please?

Reply:
@osvaldovega So Ali the text is okay